<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>700</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>930</width>
       <height>700</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/Resources/1stWall.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>630</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>930</width>
       <height>700</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/Resources/newUser.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="okBtn_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>428</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>428</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameField">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>350</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: #18d401; /* Color verde personalizado */
}
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>0</y>
       <width>930</width>
       <height>700</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/Resources/userRest.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>430</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLineEdit" name="userPost">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background: transparent; /* Sin fondo */
    border: none; /* Sin bordes */
    color: #18d401; /* Color verde personalizado */
    font-family: &quot;Uncial Antiqua&quot;; /* Tipografía decorativa */
    font-size: 18pt; /* Tamaño ajustado */
    font-weight: bold; /* Negrita para destacar */
}
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startGameBtn">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>58</y>
       <width>371</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="createUserBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>146</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-2</y>
       <width>930</width>
       <height>700</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/Resources/welcomePg.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="exitGame_3">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>590</y>
       <width>83</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>exitGame_3</zorder>
    <zorder>createUserBtn</zorder>
    <zorder>startGameBtn</zorder>
    <zorder>userPost</zorder>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="pic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>940</width>
       <height>700</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/Resources/addUser.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_newuser">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background: transparent; /* Sin fondo */
    border: none; /* Sin bordes */
    color: #18d401; /* Color verde personalizado */
    font-family: &quot;Uncial Antiqua&quot;; /* Tipografía decorativa */
    font-size: 18pt; /* Tamaño ajustado */
    font-weight: bold; /* Negrita para destacar */
}


</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>470</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>470</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>930</width>
       <height>700</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/Resources/createUser.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="createButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>410</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="renameButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>500</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>550</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>500</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>560</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListWidget" name="userListWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>531</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
    background: transparent; /* Fondo transparente */
    border: none;            /* Sin bordes */
    color: white;            /* Texto normal en blanco */
    font-family: &quot;Uncial Antiqua&quot;; /* Tipografía decorativa */
    font-size: 16pt;         /* Tamaño de fuente ajustado */
}

QListWidget::item {
    color: white; /* Texto normal en blanco */
    font-family: &quot;Uncial Antiqua&quot;; /* Tipografía decorativa */
}

QListWidget::item:selected {
    background: transparent; /* Fondo transparente para el seleccionado */
    color: #18d401;          /* Texto verde personalizado para seleccionado */
    border: 1px solid #18d401; /* Borde verde alrededor del seleccionado */
    font-family: &quot;Uncial Antiqua&quot;; /* Tipografía decorativa */
    font-weight: bold;       /* Negrita para destacar */
}

</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_9">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>930</width>
       <height>700</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/Resources/addUser.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="newUser">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: #18d401; /* Color verde personalizado */
}
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>470</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>470</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>930</width>
       <height>700</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/Resources/renameUser.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="renameConfirmButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>480</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelrename">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>480</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="rename">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>390</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: #18d401; /* Color verde personalizado */
}
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>930</width>
       <height>700</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/Resources/deleteUser.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>480</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>480</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: white; /* Opcional, si deseas cambiar el color del texto */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
